test/suites/dev/moonbase/test-moonbeam-lazy-migrations/test-unlock-democracy-funds.ts

```diff
@@ -0,0 +1,1638 @@
+        data:
+          "0x0004540000000080000064a7b3b6e00d00000000000000000000000000000000000000000000000000" +
+          "0000000000000000000000000000000000000000000000000000000000000000000000",
+      },
+      {
+        key:
+          "0xf2794c22e353e9a839f12faab03a911be470c6afbbbc027eb288ade7595953c207ef3c8caa332c3e602" +
+          "604c910870a7c5c189f6c0ab60d6f13c5880d",
+        data:
+          "0x00043300000000000000f4448291634500000000000000000000000000000000000000000000000000" +
+          "0000000000000000000000000000000000000000000000000000000000000000000000",
+      },
+      {
+        key:
+          "0xf2794c22e353e9a839f12faab03a911be470c6afbbbc027eb288ade7595953c207f018e75b92d78ef4d" +
+          "e4067706feb9d81518812d41a1542b3c36bc5",
+        data:
+          "0x0000000000000000000000000000000000000000000000000000000000000000000000000000000000" +
+          "00000000000000000000000000",
+      },
+    ];
+
+    it({
+      id: "T01",
+      title: "Test unlockDemocracyFunds drains democracy votingOf and also unlocks the funds.",
+      test: async function () {
+        for (let i = 0; i < voting_storage.length; i++) {
+          await context.createBlock(
+            context
+              .polkadotJs()
+              .tx.sudo.sudo(
+                context
+                  .polkadotJs()
+                  .tx.system.setStorage([[voting_storage[i]["key"], voting_storage[i]["data"]]])
+              )
+              .signAsync(alith)
+          );
+        }
+        expect((await context.polkadotJs().query.democracy.votingOf.entries()).length).is.equal(
+          100
+        );
+
+        for (let i = 0; i < balance_storage.length; i++) {
+          await context.createBlock(
+            context
+              .polkadotJs()
+              .tx.sudo.sudo(
+                context
+                  .polkadotJs()
+                  .tx.system.setStorage([[balance_storage[i]["key"], balance_storage[i]["data"]]])
+              )
+              .signAsync(alith)
+          );
+        }
+        // 100 just added + alith
+        expect((await context.polkadotJs().query.balances.locks.entries()).length).is.equal(101);
+
+        const tx1 = await context.createBlock(
+          await context.polkadotJs().tx.moonbeamLazyMigrations.unlockDemocracyFunds(30)
+        );
+        expect(tx1.result?.successful, "Failed to unlock democracy funds");
+        expect((await context.polkadotJs().query.democracy.votingOf.entries()).length).is.equal(70);
+
+        /// Check that the locks are removed for the first 30 accounts
+        for (let i = 0; i < 30; i++) {
+          const locks = await context
+            .polkadotJs()
+            .query.balances.locks(balance_storage[i]["account"]);
+          for (let j = 0; j < locks.length; j++) {
+            expect(locks[j].id.toHuman()).is.not.equal("democrac");
+          }
+        }
+
+        const tx2 = await context.createBlock(
```

Comment:
Each check should be on a separate `it` block

Reply:
Done. Hopefully I understood correctly.

Reply:
I've asked @timbrinded and since this is a "complex" test where we need to do multiple sequential things, it should be in 1 single test.
However, I did extract the one checking the upper limit.